src/main.sh

```diff
@@ -4,3 +4,7 @@ echo "Install programs.."
 
 IFS=" " read -r -a programs <<< "$(find ./src/programs -type f | sed 's!.*/!!' | tr '\n' ' ')"
 sudo xbps-install -y "${programs[@]}"
+
+
+echo "Download dotfiles"
+git clone --bare https://github.com/balintkiraly/.dotfiles.git $HOME/.dotfiles
```

Comment:
**[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Double quote to prevent globbing and word splitting. [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)